modules/dag-l1/src/main/scala/org/tessellation/dag/l1/Main.scala

```diff
@@ -33,7 +34,7 @@ object Main extends TessellationIOApp[Run]("", "DAG L1 node", version = BuildInf
     Database.forAsync[IO](cfg.db).flatMap { implicit database =>
       for {
         queues <- Queues.make[IO](sdkQueues).asResource
-        storages <- Storages.make[IO](cfg.tips, sdkStorages, method.l0Peer).asResource
+        storages <- Storages.make[IO](cfg.tips, sdkStorages, method.l0Peer, SnapshotOrdinal.MinValue).asResource
```

Comment:
After implementing download/redownload, this will be verified. Probably we will be passing the ordinal fetched during download here.